src/ts/uimanager.ts

```diff
@@ -763,8 +763,23 @@ class PlayerWrapper {
     for (let property of properties) {
       // Get an eventually existing property descriptor to differentiate between plain properties and properties with
       // getters/setters.
-      let propertyDescriptor: PropertyDescriptor = Object.getOwnPropertyDescriptor(player, property) ||
-        Object.getOwnPropertyDescriptor(Object.getPrototypeOf(player), property);
+      const getPropertyDescriptor = (object: PlayerAPI): PropertyDescriptor => {
```

Comment:
`object` is a keyword in TypeScript so better to avoid that (wasn't used when the code was written, but is now).

Also just fyi this would be the iterative non-recursive way:
```ts
const propertyDescriptor = ((target: object) => {
    while (target) {
        const propertyDescriptor = Object.getOwnPropertyDescriptor(target, property);
        if (propertyDescriptor) {
            return propertyDescriptor;
        }
        target = Object.getPrototypeOf(target);
    }
})(player);
```

Reply:
I prefer the non-recursive way as it's much cleaner.